libminifi/src/provenance/Provenance.cpp

```diff
@@ -45,13 +45,13 @@ const char *ProvenanceEventRecord::ProvenanceEventTypeStr[REPLAY + 1] = { "CREAT
 
 ProvenanceEventRecord::ProvenanceEventRecord(ProvenanceEventRecord::ProvenanceEventType event, std::string componentId, std::string componentType)
     : core::SerializableComponent(core::getClassName<ProvenanceEventRecord>()),
-      _entryDate(0),
-      _lineageStartDate(0),
+      _entryDate(std::chrono::time_point<std::chrono::system_clock>()),
```

Comment:
is there any advantage using this over `std::chrono::system_clock::time_point`? the latter is not much shorter so I don't insist on changing it, just asking

Reply:
We could also just delete the line and let the default constructor be called automatically.

Reply:
Removed these and other unneccesary initializations from the ctors in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3

I prefer the `std::chrono::system_clock::time_point` aswell, its more readable thanks for idea @adamdebreceni, changed all references of time_point<x_clock> to x_clock::time_point in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3